contracts/src/core/libraries/v3/access-control/AccessControlConstants.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+library AccessControlConstants {
+    bytes32 internal constant OWNER_ROLE = keccak256("OWNER");
+    bytes32 internal constant QUORUM_OWNER_SEED = keccak256("QUORUM_OWNER");
+
+    function QUORUM_OWNER_ROLE(uint64 quorumId) internal pure returns (bytes32) {
```

Comment:
how should this be used? Is the idea that the caller would call
```
EigenDAAccessControl.setupRole(AccessControlConstants.QUORUM_OWNER_ROLE(quorumId))
```
? Or how else is this supposed to be used? should we add a quorum_setter in EigenDAAccessControl already?

Reply:
This intended use case should probably be documented in the contract then.